jargons/webpack.md

```diff
@@ -4,4 +4,13 @@ tags:
 - javascript
 ---
 
-[Node.js](/nodejs) ile yazılan kodun tarayıcıda da çalışabilmesi için gerekli olan bir araçtır.
+En basit anlatımıyla; [Node.js](/nodejs) ile javascript üzerindeki bağımlılıklarınızı yöneten, derleyen ve birleştiren araçtır, bunun yanı sıra eklentileri ile birlikte pek çok görev otomasyonu için kullanılabilir.
```

Comment:
Grunt ve Gulp ile karşılaştırınca, Webpack ile yapılabileceklerin pek de sınırı yok. Burdaki örnekler, yukarıdaki Gulp ve Grunt'tan kopyalanmış olmasının yanı sıra, Webpack'i onlarla aynı kefeye koyuyor. 
Basit haliyle, Webpack bir `module bundler`, Grunt ve Gulp ise `task runner`. @unexge de aynı şeyi belirtmiş zaten. 

Bana kalırsa bu jargonda yapılan değişiklik iptal edilip, daha sonra Webpack için Grunt ve Gulp'tan tamamen bağımsız bir açıklama yazılması daha doğru olacaktır. 

Son olarak, eğer bu haliyle değiştirilmesi kabul edilirse, yukardaki durumlar burda da geçerli: HTML, JavaScript, CSS, görüntü, external link.